.github/workflows/test-linux-qt6.yml

```diff
@@ -0,0 +1,160 @@
+name: Linux tests with PyQt6/PySide6
+
+on:
+  push:
+    branches:
+      - master
+      - qt6_ci # for testing only
```

Comment:
Using a cron-based trigger is probably more sensible for now?

The `qt6_ci` is for testing only and will be removed